pkg/pool-stable/test/ComposableStablePool.test.ts

```diff
@@ -378,9 +390,12 @@ describe('ComposableStablePool', () => {
 
         it('returns the pre-join invariant', async () => {
           const { value, precision } = await pool.getAmplificationParameter();
+
+          // We pass a floating point amplification instead of an integer one, to more closely match the behavior in the
+          // Pool.
           const expectedInvariant = calculateInvariant(
             registeredBalances.filter((_, i) => i !== bptIndex),
-            value.div(precision)
+            value.toNumber() / precision.toNumber()
```

Comment:
Note: These are not in the diff in https://github.com/balancer-labs/balancer-v2-monorepo/pull/2059, but were present in the base already.